src/telliot_feed_examples/sources/blockhash_aggregator.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+from datetime import datetime
+from datetime import timezone
+from typing import Any
+
+import requests
+from telliot_core.datasource import DataSource
+from telliot_core.dtypes.datapoint import DataPoint
+from web3 import Web3
+
+from telliot_feed_examples.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+w3 = Web3(
```

Comment:
Use the `Web3` instance from `TelliotConfig` instead. For example, something like:
```python
cfg = TelliotConfig()
print('latest block num', cfg.get_endpoint().web3.eth.get_block("latest")
```